services/console-proxy/server/src/main/java/com/cloud/consoleproxy/vnc/network/NioSocketHandler.java

```diff
@@ -20,6 +20,8 @@
 
 public interface NioSocketHandler {
 
+    int DEFAULT_BUF_SIZE = 16 * 1024;
```

Comment:
is there a reason we need to keep this hard coded? might be a configurable to allow for tuning by the operator, no?

Reply:
It's just refactoring, it used the same number 16384 in other places, but didn't use it consistently across all stream usage.